detekt-config.yml

```diff
@@ -595,7 +595,7 @@ style:
   RedundantVisibilityModifierRule:
     active: false
   ReturnCount:
-    active: true
+    active: false
```

Comment:
This is the main thing I wanted to update

Reply:
Good call.